src/librustdoc/visit_ast.rs

```diff
@@ -130,7 +131,7 @@ impl<'a, 'tcx> RustdocVisitor<'a, 'tcx> {
     fn visit_mod_contents(
         &mut self,
         span: Span,
-        vis: &'tcx hir::Visibility<'_>,
```

Comment:
Do you want to change this to take the visibility by value since it is Copy?

Reply:
The visibility is not Copy:
```
error[E0507]: cannot move out of `item.vis` which is behind a shared reference
   --> src/librustdoc/visit_ast.rs:316:65
    |
316 |                 om.mods.push(self.visit_mod_contents(item.span, item.vis, item.hir_id(), m, name));
    |                                                                 ^^^^^^^^ move occurs because `item.vis` has type `Spanned<rustc_hir::VisibilityKind<'_>>`, which does not implement the `Copy` trait
```

Reply:
Ah, I got confused because `clean::Visibility` is Copy.